utils/verify_intel_gaudi/roles/verify_intel_gaudi/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+- name: Initialise node accelerator status
+  ansible.builtin.set_fact:
+    node_has_accelerator: false
+
+- name: Check node has Gaudi2 or not
+  ansible.builtin.shell: |
+    set -o pipefail
+    lspci -n -d 1da3:1020:1200
```

Comment:
Can we use same lspci check used in accelerators and other tasks

Reply:
this is to detect Gaudi2 in particular (we don't to run this on Gaudi 3 for the moment

Reply:
changed to a variable